bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -476,11 +482,13 @@ async fn extension_loop_active(
                                     );
                                     lambda_enhanced_metrics.set_report_log_metrics(&metrics);
                                     let mut p = invocation_processor.lock().await;
-                                    if let Some((post_runtime_duration_ms, network_offset)) = p.on_platform_report(&request_id, metrics.duration_ms) {
-                                        lambda_enhanced_metrics.set_post_runtime_duration_metric(
-                                            post_runtime_duration_ms,
-                                        );
-                                        lambda_enhanced_metrics.set_network_enhanced_metrics(network_offset);
+                                    let (post_runtime_duration_ms, enhanced_metric_data) = p.on_platform_report(&request_id, metrics.duration_ms);
+                                    if let Some(duration) = post_runtime_duration_ms {
+                                        lambda_enhanced_metrics.set_post_runtime_duration_metric(duration);
+                                    }
+                                    if let Some(offsets) = enhanced_metric_data {
+                                        lambda_enhanced_metrics.set_network_enhanced_metrics(offsets.network_offset);
+                                        lambda_enhanced_metrics.set_cpu_time_enhanced_metrics(offsets.cpu_offset);
                                     }
                                     drop(p);
```

Comment:
Not sure I understand why are we not doing the some as before?

Reply:
I realized that we were returning None for both `post_runtime_duration_ms` and `enhanced_metric_data` if we could not calculate `post_runtime_duration_ms` because of `context.runtime_duration_ms` being 0. Changed it to handle so that even if we can't calculate `post_runtime_duration_ms`, we still try to send enhanced metrics.